flink-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/HBaseTableFactory.java

```diff
@@ -119,7 +126,17 @@ private HBaseTableSchema validateTableSchema(TableSchema schema) {
 				String[] qualifierNames = familyType.getFieldNames();
 				TypeInformation[] qualifierTypes = familyType.getFieldTypes();
 				for (int j = 0; j < familyType.getArity(); j++) {
-					hbaseSchema.addColumn(name, qualifierNames[j], qualifierTypes[j].getTypeClass());
+					// HBase connector doesn't support LocalDateTime
+					// use Timestamp as conversion class for now.
+					Class clazz = qualifierTypes[j].getTypeClass();
+					if (LocalDateTime.class.equals(clazz)) {
+						clazz = Timestamp.class;
```

Comment:
Not work for nested fields?

Reply:
You are right, currently HBase connector only support atomic types as columns (HBase qualifier).